core/src/components/dropdown-v2/readme.md

```diff
@@ -1,6 +1,6 @@
-# sdds-dropdown-v2
+# tds-dropdown-v2
 
-The dropdown can be populated with options either by passing the `<sdds-dropdown-option>` as a child to the element. Or by specifying a data property on the element.
+The dropdown can be populated with options either by passing the `<tds-dropdown-option>` as a child to the element. Or by specifying a data property on the element.
 This property should be a JSON array containing objects of the following structure:
 
 ```json
```

Comment:
Maybe remove this part from the readme?

Reply:
Since this is still a very beta component we should wait with extensive docs until we know how the API is gonna look like